server/controller/users.js

```diff
@@ -24,15 +24,16 @@ class UserController {
    */
   static async registerUser(req, res, next) {
     const {
-      firstName, lastName, email, userName, password, gender
+      firstName, lastName, email, role, userName, password, gender
     } = req.body;
-
+    const userRole = role ? role.toLowerCase() : 'trainee';
```

Comment:
We agreed to get rid of this here and just set the `role` to a default value of `trainee` on the User model @chekwas88